src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/AbstractInversionConfiguration.java

```diff
@@ -4,25 +4,30 @@
 import java.util.List;
 
 import org.dom4j.Element;
+import org.opensha.commons.data.uncertainty.UncertainIncrMagFreqDist;
 import org.opensha.commons.metadata.XMLSaveable;
 import org.opensha.commons.util.XMLUtils;
 import org.opensha.sha.magdist.IncrementalMagFreqDist;
 
-import scratch.UCERF3.inversion.InversionTargetMFDs;
-import scratch.UCERF3.inversion.UCERF3InversionConfiguration.SlipRateConstraintWeightingType;
-import scratch.UCERF3.utils.MFD_InversionConstraint;
-import scratch.UCERF3.utils.MFD_WeightedInversionConstraint;
+import org.opensha.sha.earthquake.faultSysSolution.modules.InversionTargetMFDs;
 
 public class AbstractInversionConfiguration implements XMLSaveable  {
 
+	public static enum NZConstraintWeightingType{
+		NORMALIZED,
+		UNNORMALIZED,
+		BOTH,
+		NORMALIZED_BY_UNCERTAINTY
```

Comment:
@voj as dicsussed, I do prefer this name , but we need some backwards compatility to support "prior ENUM" (UNCERTAINTY_WEIGHTED)